src/test/java/dev/snowdrop/release/services/IssueServiceTest.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package dev.snowdrop.release.services;
+
+import com.atlassian.jira.rest.client.api.JiraRestClient;
+import com.atlassian.jira.rest.client.api.domain.BasicIssue;
+import com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import dev.snowdrop.release.model.Issue;
+import dev.snowdrop.release.model.POM;
+import dev.snowdrop.release.model.Release;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author <a href="antcosta@redhat.com">Antonio Costa</a>
+ */
+@QuarkusTest
+@TestProfile(TestProfiles.IntegrationTests.class)
+public class IssueServiceTest {
+
+    private static final YAMLMapper MAPPER = new YAMLMapper();
+
+    @Inject
+    JiraIssueFactory factory;
+
+
+    @Inject
+    IssueService service;
+
+    private static JiraRestClient client(final String jiraUser, final String jiraPw) {
```

Comment:
The client is built in the `JiraClientConfigurationProperties`, this method is never used

Reply:
done